src/textual/app.py

```diff
@@ -326,14 +326,14 @@ class MyApp(App[None]):
     """
 
     ENABLE_COMMAND_PALETTE: ClassVar[bool] = True
-    """Should the [command palette][textual.command_palette.CommandPalette] be enabled for the application?"""
+    """Should the [command palette][textual.command.CommandPalette] be enabled for the application?"""
 
-    COMMAND_SOURCES: ClassVar[set[type[CommandSource]]] = {SystemCommandSource}
-    """The [command sources](/api/command_palette/) for the application.
+    COMMAND_SOURCES: ClassVar[set[type[Source]]] = {SystemCommandSource}
+    """The [command sources](/api/command/) for the application.
```

Comment:
We should use paths to textual objects because it's easier to keep URLs synced that way.
The `[some text here][textual.module.Object]` ones.